innovation_sweet_spots/analysis/wrangling_utils.py

```diff
@@ -409,6 +409,10 @@ def gtr_topics(self):
             self._gtr_topics = gtr.get_gtr_topics().rename(columns={"text": "topic"})
         return self._gtr_topics
 
+    @property
+    def gtr_topics_list(self):
+        return sorted(self.gtr_topics.topic.to_list())
```

Comment:
There's no docstring for this function (maybe not needed, as the function is straightforward 🙂)

Reply:
Thanks will add a docstring for a good measure
```suggestion
    def gtr_topics_list(self):
        """Returns a sorted list of GtR research topics (categories)"""
        return sorted(self.gtr_topics.topic.to_list())
```